recipes/perl-carp/meta.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: perl-carp
+  version: "1.36"
+
+#source:
+  #fn: 
+  #url: 
+  #md5: 
```

Comment:
The conda skeleton has some interesting ideas.

Anything like this is a core perl module, so its called as a dependency, but there's nothing to download because its already there.